### PR TITLE
Change acct creation steps order and in-linedness

### PR DESCRIPTION
**Pull request by atz**
Address error during account creation. 

See: https://app.honeybadger.io/projects/48327/faults/33545313#notice-trace

### CODE REVIEW

File: app/services/create_account.rb [hunk 1]
```diff
@@ -13,9 +13,12 @@ def save
...
   end
 
+  # `Apartment::Tenant.create` calls the DB adapter's `switch`, which we have a hook into
+  # via an initializer.  In our hook we do `account.switch!` and that requires a well-formed
+  # Account (i.e. creation steps complete, endpoints populated).  THEREFORE, `create_tenant`
+  # must be called *after* all external resources are provisioned.
   def create_external_resources
-    create_tenant &&
-      create_account_inline
+    create_account_inline && create_tenant
```
Comment (darrenleeweber) [inline app/services/create_account.rb]: How do the jobs get the tenant UUID if it doesn't exist yet?

**Comment by mjgiarlo** [comment]
@atz I tested this locally, and I can't get into newly created tenants because this error triggers: https://github.com/projecthydra-labs/hyku/blob/master/app/controllers/application_controller.rb#L26-L28

**Comment by mjgiarlo** [comment]
Rails console log:
```
Started POST "/account/sign_up?locale=en" for 127.0.0.1 at 2017-05-11 09:30:52 -0700
  Account Load (0.3ms)  SELECT  "public"."accounts".* FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 LIMIT $2  [["cname", "localhost"
], ["LIMIT", 1]]
Processing by AccountSignUpController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"NJ/jsm8P7QPuLKOpR0WiX0AIcCmSBf+kjmsqt9D/Zdgj/g+SCZTjp7Jmua1Cv6hHf6eGKluf4I9Y97sPilVN5A==", "account
"=>{"name"=>"quux"}, "commit"=>"Save", "locale"=>"en", "host"=>"localhost"}
  Site Load (0.3ms)  SELECT  "sites".* FROM "sites" ORDER BY "sites"."id" ASC LIMIT $1  [["LIMIT", 1]]
Usergroups are ["public"]
  Hyrax::Feature Load (0.4ms)  SELECT  "hyrax_features".* FROM "hyrax_features" WHERE "hyrax_features"."key" = $1 ORDER BY "hyrax_features"."id" ASC L
IMIT $2  [["key", "transfer_works"], ["LIMIT", 1]]
  Hyrax::Feature Load (0.4ms)  SELECT  "hyrax_features".* FROM "hyrax_features" WHERE "hyrax_features"."key" = $1 ORDER BY "hyrax_features"."id" ASC L
IMIT $2  [["key", "proxy_deposit"], ["LIMIT", 1]]
  CACHE (0.0ms)  SELECT  "public"."accounts".* FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 LIMIT $2  [["cname", "localhost"], ["LI
MIT", 1]]
   (0.1ms)  BEGIN
  Account Exists (0.3ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "b87e4b7c-992e-4729-
9a4e-dc8271c9f67c"], ["LIMIT", 1]]
  Account Exists (0.2ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 LIMIT $2  [["cname", "quux.localhost"], ["L
IMIT", 1]]
  SQL (0.3ms)  INSERT INTO "public"."accounts" ("tenant", "cname", "created_at", "updated_at", "name") VALUES ($1, $2, $3, $4, $5) RETURNING "id"  [["
tenant", "b87e4b7c-992e-4729-9a4e-dc8271c9f67c"], ["cname", "quux.localhost"], ["created_at", 2017-05-11 16:30:52 UTC], ["updated_at", 2017-05-11 16:3
0:52 UTC], ["name", "quux"]]
   (3.0ms)  COMMIT
  Account Load (0.3ms)  SELECT  "public"."accounts".* FROM "public"."accounts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"]
, ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] Performing CreateAccountInlineJob from Async(default) with arguments: #<Gl
obalID:0x007ff18d8a2d08 @uri=#<URI::GID gid://hyku/Account/4>>
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.2ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352] Performin
g CreateSolrCollectionJob from Async(default) with arguments: #<GlobalID:0x007ff18d8a2d08 @uri=#<URI::GID gid://hyku/Account/4>>
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352]    (0.2ms
)  BEGIN
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352]   Account
 Exists (0.4ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."tenant" = $1 AND ("public"."accounts"."id" != $2) LIMIT $3  [["t
enant", "b87e4b7c-992e-4729-9a4e-dc8271c9f67c"], ["id", 4], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352]   Account
 Exists (0.3ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 AND ("public"."accounts"."id" != $2) LIMIT $3  [["cn
ame", "quux.localhost"], ["id", 4], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352]   SQL (0.
9ms)  INSERT INTO "public"."endpoints" ("type", "options", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"  [["type", "SolrEndpoint
"], ["options", "<174 bytes of binary data>"], ["created_at", 2017-05-11 16:30:55 UTC], ["updated_at", 2017-05-11 16:30:55 UTC]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352]   SQL (0.
5ms)  UPDATE "public"."accounts" SET "updated_at" = $1, "solr_endpoint_id" = $2 WHERE "public"."accounts"."id" = $3  [["updated_at", 2017-05-11 16:30:
55 UTC], ["solr_endpoint_id", 1], ["id", 4]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352]    (3.3ms
)  COMMIT
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateSolrCollectionJob] [298f510e-3828-4641-8483-cbfbd2d6e352] Performed
 CreateSolrCollectionJob from Async(default) in 2867.99ms
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.2ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.2ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f] Performin
g CreateFcrepoEndpointJob from Async(default) with arguments: #<GlobalID:0x007ff18d8a2d08 @uri=#<URI::GID gid://hyku/Account/4>>
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f]    (0.1ms
)  BEGIN
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f]   Account
 Exists (0.3ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."tenant" = $1 AND ("public"."accounts"."id" != $2) LIMIT $3  [["t
enant", "b87e4b7c-992e-4729-9a4e-dc8271c9f67c"], ["id", 4], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f]   Account
 Exists (0.3ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 AND ("public"."accounts"."id" != $2) LIMIT $3  [["cn
ame", "quux.localhost"], ["id", 4], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f]   SQL (0.
3ms)  INSERT INTO "public"."endpoints" ("type", "options", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"  [["type", "FcrepoEndpoi
nt"], ["options", "<107 bytes of binary data>"], ["created_at", 2017-05-11 16:30:55 UTC], ["updated_at", 2017-05-11 16:30:55 UTC]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f]   SQL (0.
5ms)  UPDATE "public"."accounts" SET "updated_at" = $1, "fcrepo_endpoint_id" = $2 WHERE "public"."accounts"."id" = $3  [["updated_at", 2017-05-11 16:3
0:55 UTC], ["fcrepo_endpoint_id", 2], ["id", 4]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f]    (3.2ms
)  COMMIT
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateFcrepoEndpointJob] [f261e58a-1d7e-4969-837f-06bee8779b7f] Performed
 CreateFcrepoEndpointJob from Async(default) in 13.07ms
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.2ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.6ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e] Performin
g CreateRedisNamespaceJob from Async(default) with arguments: #<GlobalID:0x007ff18d8a2d08 @uri=#<URI::GID gid://hyku/Account/4>>
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e]    (0.1ms
)  BEGIN
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e]   Account
 Exists (0.4ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."tenant" = $1 AND ("public"."accounts"."id" != $2) LIMIT $3  [["t
enant", "b87e4b7c-992e-4729-9a4e-dc8271c9f67c"], ["id", 4], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e]   Account
 Exists (0.3ms)  SELECT  1 AS one FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 AND ("public"."accounts"."id" != $2) LIMIT $3  [["cn
ame", "quux.localhost"], ["id", 4], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e]   SQL (0.
3ms)  INSERT INTO "public"."endpoints" ("type", "options", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"  [["type", "RedisEndpoin
t"], ["options", "<104 bytes of binary data>"], ["created_at", 2017-05-11 16:30:55 UTC], ["updated_at", 2017-05-11 16:30:55 UTC]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e]   SQL (0.
4ms)  UPDATE "public"."accounts" SET "updated_at" = $1, "redis_endpoint_id" = $2 WHERE "public"."accounts"."id" = $3  [["updated_at", 2017-05-11 16:30
:55 UTC], ["redis_endpoint_id", 3], ["id", 4]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e]    (3.2ms
)  COMMIT
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateRedisNamespaceJob] [540e67df-5d7c-4182-a8ac-24a7a5cb439e] Performed
 CreateRedisNamespaceJob from Async(default) in 12.17ms
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.1ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.1ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateDefaultAdminSetJob] [2462c3af-a252-4b2b-a504-565f24651902] Performi
ng CreateDefaultAdminSetJob from Async(default) with arguments: #<GlobalID:0x007ff18d8a2d08 @uri=#<URI::GID gid://hyku/Account/4>>
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateDefaultAdminSetJob] [2462c3af-a252-4b2b-a504-565f24651902]   Accoun
t Load (0.2ms)  SELECT  "public"."accounts".* FROM "public"."accounts" WHERE "public"."accounts"."cname" = $1 LIMIT $2  [["cname", "quux.localhost"],
["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] [CreateDefaultAdminSetJob] [2462c3af-a252-4b2b-a504-565f24651902] Performe
d CreateDefaultAdminSetJob from Async(default) in 1.81ms
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d]   Account Load (0.1ms)  SELECT  "public"."accounts".* FROM "public"."accou
nts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"], ["LIMIT", 1]]
[ActiveJob] [CreateAccountInlineJob] [8245423b-e5ae-4f90-a2d7-a04b7c92465d] Performed CreateAccountInlineJob from Async(default) in 2916.11ms
  Account Load (0.1ms)  SELECT  "public"."accounts".* FROM "public"."accounts" WHERE "public"."accounts"."tenant" = $1 LIMIT $2  [["tenant", "public"]
, ["LIMIT", 1]]
Completed 404 Not Found in 2963ms (ActiveRecord: 27.4ms)


  
ActionController::RoutingError (Not Found):
  
app/controllers/application_controller.rb:27:in `block in <class:ApplicationController>'
  Rendering /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/rescues/routing_error.html.erb within r
escues/layout
  Rendering /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb
  Rendered /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb (1.6ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [14
6 times] (32.2ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [4 
times] (0.9ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [7 
times] (1.4ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [9 
times] (3.2ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [15
8 times] (32.0ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [1 
times] (0.4ms)
  Rendered collection of /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_route.html.erb [3 
times] (1.3ms)
  Rendered /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/routes/_table.html.erb (4.4ms)
  Rendering /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/rescues/_request_and_response.html.erb
  Rendered /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/rescues/_request_and_response.html.erb (
1.0ms)
  Rendered /home/mjg/.rvm/gems/ruby-2.3.3@hyku/gems/actionpack-5.0.2/lib/action_dispatch/middleware/templates/rescues/routing_error.html.erb within re
scues/layout (135.0ms)
```

**Comment by mjgiarlo** [comment]
Traceback:

```
app/controllers/application_controller.rb:27:in `block in <class:ApplicationController>'
activesupport (5.0.2) lib/active_support/rescuable.rb:107:in `instance_exec'
activesupport (5.0.2) lib/active_support/rescuable.rb:107:in `block in handler_for_rescue'
activesupport (5.0.2) lib/active_support/rescuable.rb:89:in `rescue_with_handler'
activesupport (5.0.2) lib/active_support/rescuable.rb:158:in `rescue_with_handler'
actionpack (5.0.2) lib/action_controller/metal/rescue.rb:23:in `rescue in process_action'
actionpack (5.0.2) lib/action_controller/metal/rescue.rb:20:in `process_action'
actionpack (5.0.2) lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
activesupport (5.0.2) lib/active_support/notifications.rb:164:in `block in instrument'
activesupport (5.0.2) lib/active_support/notifications/instrumenter.rb:21:in `instrument'
activesupport (5.0.2) lib/active_support/notifications.rb:164:in `instrument'
actionpack (5.0.2) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
actionpack (5.0.2) lib/action_controller/metal/params_wrapper.rb:248:in `process_action'
activerecord (5.0.2) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
actionpack (5.0.2) lib/abstract_controller/base.rb:126:in `process'
actionview (5.0.2) lib/action_view/rendering.rb:30:in `process'
actionpack (5.0.2) lib/action_controller/metal.rb:190:in `dispatch'
actionpack (5.0.2) lib/action_controller/metal.rb:262:in `dispatch'
actionpack (5.0.2) lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
actionpack (5.0.2) lib/action_dispatch/routing/route_set.rb:32:in `serve'
actionpack (5.0.2) lib/action_dispatch/journey/router.rb:39:in `block in serve'
actionpack (5.0.2) lib/action_dispatch/journey/router.rb:26:in `each'
actionpack (5.0.2) lib/action_dispatch/journey/router.rb:26:in `serve'
actionpack (5.0.2) lib/action_dispatch/routing/route_set.rb:725:in `call'
peek-performance_bar (1.2.1) lib/peek/views/performance_bar/process_utilization.rb:144:in `call'
is_it_working (1.1.0) lib/is_it_working/handler.rb:48:in `call'
apartment (1.2.0) lib/apartment/elevators/generic.rb:24:in `call'
apartment (1.2.0) lib/apartment/reloader.rb:18:in `call'
rack (2.0.2) lib/rack/deflater.rb:34:in `call'
warden (1.2.7) lib/warden/manager.rb:36:in `block in call'
warden (1.2.7) lib/warden/manager.rb:35:in `catch'
warden (1.2.7) lib/warden/manager.rb:35:in `call'
rack (2.0.2) lib/rack/etag.rb:25:in `call'
rack (2.0.2) lib/rack/conditional_get.rb:38:in `call'
rack (2.0.2) lib/rack/head.rb:12:in `call'
rack (2.0.2) lib/rack/session/abstract/id.rb:232:in `context'
rack (2.0.2) lib/rack/session/abstract/id.rb:226:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/cookies.rb:613:in `call'
active-fedora (11.1.6) lib/active_fedora/ldp_cache.rb:26:in `call'
flipflop (2.3.1) lib/flipflop/feature_cache.rb:12:in `call'
activerecord (5.0.2) lib/active_record/migration.rb:553:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.2) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.2) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.2) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.2) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
web-console (3.5.0) lib/web_console/middleware.rb:135:in `call_app'
web-console (3.5.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (3.5.0) lib/web_console/middleware.rb:18:in `catch'
web-console (3.5.0) lib/web_console/middleware.rb:18:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.2) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.2) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.2) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.2) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.2) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.2) lib/rails/rack/logger.rb:24:in `call'
sprockets-rails (3.2.0) lib/sprockets/rails/quiet_assets.rb:13:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.2) lib/rack/method_override.rb:22:in `call'
rack (2.0.2) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.2) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.2) lib/rack/sendfile.rb:111:in `call'
secure_headers (3.6.4) lib/secure_headers/middleware.rb:12:in `call'
honeybadger (3.1.2) lib/honeybadger/rack/error_notifier.rb:34:in `block in call'
honeybadger (3.1.2) lib/honeybadger/agent.rb:321:in `with_rack_env'
honeybadger (3.1.2) lib/honeybadger/rack/error_notifier.rb:31:in `call'
honeybadger (3.1.2) lib/honeybadger/rack/user_feedback.rb:31:in `call'
honeybadger (3.1.2) lib/honeybadger/rack/user_informer.rb:21:in `call'
railties (5.0.2) lib/rails/engine.rb:522:in `call'
puma (3.8.2) lib/puma/configuration.rb:224:in `call'
puma (3.8.2) lib/puma/server.rb:600:in `handle_request'
puma (3.8.2) lib/puma/server.rb:435:in `process_client'
puma (3.8.2) lib/puma/server.rb:299:in `block in run'
puma (3.8.2) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
```

**Comment by mjgiarlo** [comment]
Shows in UI as Routing Error (Not Found)

File: app/services/create_account.rb (hunk 1 above)
Comment (atz) [inline app/services/create_account.rb]: UUID exists as part of the Account model.  It is not coined by tenant schema creation.

**Comment by atz** [comment]
@mjgiarlo See if that helps

File: app/services/create_account.rb (hunk 1 above)
Comment (darrenleeweber) [inline app/services/create_account.rb]: Yea, I looked it up again and found the `self.tenant ||= SecureRandom.uuid` or similar somewhere.  For some reason, I was thinking this was created by apartment.  No problem.

**Pull request merged**